nemo/collections/tts/helpers/helpers.py

```diff
@@ -91,17 +91,7 @@ def get_batch_size(train_dataloader):
 
 
 def get_num_workers(trainer):
-    if trainer.accelerator is None:
-        return trainer.num_devices or 1
-    elif trainer.accelerator == "ddp_cpu":
-        return trainer.num_devices * trainer.num_nodes
-    elif trainer.accelerator == "ddp":
-        return trainer.num_devices * trainer.num_nodes
-    else:
-        logging.warning(
-            f"The lightning trainer received accelerator: {trainer.accelerator}. We " "recommend to use 'ddp' instead."
-        )
-        return trainer.num_devices * trainer.num_nodes
+    return trainer.num_devices * trainer.num_nodes
```

Comment:
Can you check if trainer is none and return 1?

Reply:
where would we use `get_num_workers` is trainer is None?

Reply:
Looking at the usage, both assume the trainer exists:
https://github.com/NVIDIA/NeMo/blob/b458394830719c1489261185ef728d0a2a5767a0/nemo/collections/tts/models/hifigan.py
https://github.com/NVIDIA/NeMo/blob/66ac0b3cc10d50a46635717bcfccb12426b8cefc/nemo/collections/tts/models/univnet.py

Reply:
Alright if trainer is mandatory then this is fine as is.